### ISSUE TITLE
Mapping the same field twice

### ISSUE THREAD

**Issue by blacksun1** [midknight41/map-factory#30]
Hey,

I have this and I can't remember how to overcome it

```js
const passMapper = CreateMapper();
passMapper.map('device.Id').to('device')
passMapper.map('device.id').to('deviceDetails.id');
const result = passMapper.execute({ device: { id: 1 }});
assert(result.deviceDetails.id); // Passes
assert(result.device); // Fails!!!!
```

I remember there was a hack to map the same thing twice but I can't remember what it was

[Help me Obi Wan](http://weknowmemes.com/2012/02/help-me-obi-wan/)

Regarads,

**Comment by midknight41** [comment]
Try passMapper.map(['device.id']) as it follows a different code path.

**Comment by midknight41** [comment]
I think multiple mappings should work however so it is probably a bug.

**Comment by midknight41** [comment]
@blacksun1 Published ```v1.4.1``` which fixes the bug.
